cypress/utils/table.js

```diff
@@ -33,9 +33,14 @@ function tableIsSortedBy(columnTitle) {
     .should('have.class', 'pf-c-table__sort pf-m-selected');
 }
 
+function expandAllRows() {
```

Comment:
Naming can be a bit confusing. If the button is already clicked, it will collapse the rows instead of expanding them. What about `expandCollapseAllRows()`? Or maybe even better, we define a component `rowToggler = `${TABLE_HEADER} .pf-c-table__toggle` and then instead of a function, we simple use it in the tests as `cy.get(rowToggler).click()`?

Reply:
@ikerreyes, yes, why not, I think it looks better